product/roundhouse/infrastructure.app/logging/log4net.config.xml

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0"?>
+<!-- For colors that can be used with ManagedColoredConsoleAppender, see
+     http://msdn.microsoft.com/en-us/library/system.consolecolor(v=vs.110).aspx 
+-->
 <log4net>
-  <appender name="ColoredConsoleAppender" type="log4net.Appender.ColoredConsoleAppender">
+  <appender name="ColoredConsoleAppender" type="log4net.Appender.ManagedColoredConsoleAppender">
     <mapping>
       <level value="ERROR" />
       <foreColor value="White, HighIntensity" />

```

Comment:
Note I'm not sure if high intensity is allowed. https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure/logging/log4net.mono.config.xml#L7-L34